sos/report/plugins/maas.py

```diff
@@ -62,13 +68,25 @@ def _is_snap_installed(self):
             return maas_pkg['pkg_manager'] == 'snap'
         return False
 
+    def check_enabled(self):
+        return super().check_enabled() or self.is_service("snap.maas.pebble")
```

Comment:
if the `maas` snap is there, then the `packages` tuple will cover this already, so we shouldn't need the extra check here. The `packages` tuple checks or both debs and snaps for Ubuntu platforms

Reply:
Awesome, thanks! I added a commit with a fix, should I squash it?

Reply:
yes please, squash the commits